sysfs-led-main.h

```diff
@@ -41,6 +42,15 @@ typedef enum {
 
   /** Step function used for emulating blinking via sw breathing */
   LED_RAMP_HARD_STEP = 2,
+
+  /* Triangle waveform */
```

Comment:
This and next entry have only one * when other start with two

Reply:
yup